core/backend/src/IModelTransformer.ts

```diff
@@ -698,28 +700,28 @@ export class IModelTransformer extends IModelExportHandler {
   }
 
   /** Recursively import all Elements and sub-Models that descend from the specified Subject */
-  public processSubject(sourceSubjectId: Id64String, targetSubjectId: Id64String): void {
+  public async processSubject(sourceSubjectId: Id64String, targetSubjectId: Id64String): Promise<void> {
     this.sourceDb.elements.getElement(sourceSubjectId, Subject); // throws if sourceSubjectId is not a Subject
     this.targetDb.elements.getElement(targetSubjectId, Subject); // throws if targetSubjectId is not a Subject
     this.context.remapElement(sourceSubjectId, targetSubjectId);
-    this.processChildElements(sourceSubjectId);
-    this.processSubjectSubModels(sourceSubjectId);
-    this.processDeferredElements();
+    await this.processChildElements(sourceSubjectId);
+    await this.processSubjectSubModels(sourceSubjectId);
+    await this.processDeferredElements();

```

Comment:
i think that if the last line of a method has `await`, it's better/clearer to use `return`. Our linter used to enforce that.

Reply:
I made the requested change.